app/views/hackathons/submissions/new.html.erb

```diff
@@ -1,25 +1,25 @@
-<% content_for :title, "Apply to list your hackathon" %>
+<% content_for :title, "List your hackathon on hackathons.hackclub.com" %>
 <% image = "https://cloud-rmjrb2y89-hack-club-bot.vercel.app/0oy1pj1mznloa96gh1sg-oiontn0tkxelxwjutwobkza.jpeg" %>
 
 <%= render "shared/full_page_card", background_url: image do %>
   <%= image_tag "flag-standalone.svg", style: "max-width: 30%; margin-bottom: var(--spacing-3)" %>
   <h1>
-    Apply to list your hackathon
+    List your hackathon on <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>.
```

Comment:
what are your thoughts on removing the trailing period?

also, overflow on mobile

![image](https://github.com/hackclub/hackathons-backend/assets/20099646/057f010a-2c6e-4cd2-a12c-299c2474f954)

Reply:
```suggestion
    List your hackathon on <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>
```

Let's do it!